java/src/jmri/configurexml/StoreXmlConfigAction.java

```diff
@@ -75,8 +75,8 @@ public static File getFileCustom(JFileChooser fileChooser) {
         // check for possible overwrite
         if (file.exists()) {
             int selectedValue = JOptionPane.showConfirmDialog(null,
-                    "File " + file.getName() + " already exists, overwrite it?",
-                    "Overwrite file?",
+                    rb.getString("FileOverwriteWarning1") + " " + file.getName() + " " + rb.getString("FileOverwriteWarning2"),
```

Comment:
This is the syntax for that: 
````
Bundle.getMessage("ErrorSensorNotFound", "Start")
````
from java/src//jmri/jmrit/roster/swing/speedprofile/SpeedProfilePanel.java  with ErrorSensorNotFound being 
````
java/src//jmri/jmrit/roster/swing/speedprofile/SpeedProfileBundle.properties:ErrorSensorNotFound = No {0} Sensor found
````